packages/opencensus-web-instrumentation-zone/src/interaction-tracker.ts

```diff
@@ -61,15 +68,16 @@ export class InteractionTracker {
         interceptingElement,
         task.eventName
       );
-      let taskZone = Zone.current;
+      let taskZone = Zone.root;
```

Comment:
Why is this changing?

Reply:
In #100 I update tracking rules and changed it to track only the interaction that can be labeled, that means that interactions related to clicks on the document are not tracked, the I found our this was a bug so the best initial value for taskZone is Zone.root